ha_reolink_rich_notifications.yaml

```diff
@@ -36,13 +36,31 @@ blueprint:
       selector:
         duration:
 
+    notification_title:
+      name: Notification Heading
+      description: Title of the notification
+      default: "{{ trigger.from_state.attributes.friendly_name }}"
+
+    notification_message:
+      name: Notification Subtext
+      description: Additional information
+      default: There is someone at the door
```

Comment:
This can be used for all reolink cameras, not just doorbells, so I think it would make sense to have the default be blank?